Remotion/ObjectBinding/Web/UI/Controls/BocReferenceValueImplementation/Rendering/BocAutoCompleteReferenceValueRenderer.cs

```diff
@@ -325,7 +325,7 @@ private void RenderEditModeValue (BocRenderingContext<IBocAutoCompleteReferenceV
 
       ValidationErrorRenderer.SetValidationErrorsReferenceOnControl (textBox, validationErrorsID, validationErrors);
 
-      renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Class, CssClassInput);
+      renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Class, CssClassInput + " " + CssClassDefinition.RemotionThemed);
```

Comment:
If we define the property on the base renderer (see BocRendererBase), we can use the property here.